### PR TITLE
Proposal practices addreadme

### PR DESCRIPTION
**Pull request by ashabrah**
## Author checklist
- [x] I'm proposing a single change in this PR.
- I believe the proposed standard:
  - [x] benefits more than it costs us and
  - [x] doesn't conflict with our people-first value.
- [x] The standard's file(s) follow(s) [the established directory structure][directory-structure].
- I provided:
  - [x] a succinct summary,
  - [x] reasoning for the proposed change,
  - [x] an example/instructions on how to embody the standard unless repealing, and
  - [x] a longer explanation if appropriate.

*Check boxes if requirement met or irrelevant.*

## First reviewer checklist
***NOTE:** Some of the author's checklist items are intentionally omitted, because they are a matter of opinion and up for discussion with the whole team.*

- [x] This PR contains a single change.
- [x] The proposed standard follows [the established directory structure][directory-structure].
- The author provided:
  - [x] a succinct summary,
  - [x] reasoning for the proposed change, and
  - [x] an example/instructions on how to embody the standard unless repealing.

*Check boxes if requirement met or irrelevant.*

[directory-structure]: ../tree/main/standards#directory-structure

### CODE REVIEW

**Review by lisacatalano — APPROVED** [review]

**Review by hs-epearson — DISMISSED** [review]

File: standards/practices/addingReadMe.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+# Title
```
Comment (curtisblackwell) [inline standards/practices/addingReadMe.md:2]: What's the title?

File: standards/practices/addingReadMe.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+### Proposal to add a README
```
Comment (curtisblackwell) [inline standards/practices/addingReadMe.md:3]: Is this adding something to the standard or a stray note/comment or…?

File: standards/practices/addingReadMe.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Summary
+We add a README file to our codebase when we start a project
```
Comment (curtisblackwell) [inline standards/practices/addingReadMe.md:6]: Missing punctuation.

**Review by curtisblackwell — CHANGES_REQUESTED** [review]
I like the impact and simplicity of the example/instructions section.

**Review by CristiDiaz505 — APPROVED** [review]

File: standards/practices/addingReadMe.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+# Proposal to add a README
```
Comment (curtisblackwell) [inline standards/practices/addingReadMe.md:2]: Can we call this "Adding READMEs" instead? It'll be more consistent with existing standards and the PR is a proposal, but once it's accepted and merged, this document will be a standard.

Can you also rename the file to `adding-readmes` for consistency?

I know these are somewhat tedious requests, but the more the standards grow, the more important consistency will become.

**Review by curtisblackwell — CHANGES_REQUESTED** [review]

**Review by curtisblackwell — APPROVED** [review]

**Review by sucitivel — APPROVED** [review]

**Pull request merged**